### ISSUE TITLE
Fix design on Manage Garden view

### ISSUE THREAD

**Issue by marykatefain** [HarvestHub/GardenHub#81]
See screenshot:

![screenshot from 2018-01-11 18 14 50](https://user-images.githubusercontent.com/17955536/34852183-60100658-f6fb-11e7-9d7f-e5ecc39d600c.png)

**Comment by marykatefain** [comment]
Also happening on user profile screen: 

![screenshot from 2018-01-11 19 12 20](https://user-images.githubusercontent.com/17955536/34853728-61dc6168-f703-11e7-9297-7b6886dfb66b.png)

**Comment by alexgleason** [comment]
![screen shot 2018-01-11 at 19 23 43](https://user-images.githubusercontent.com/3639540/34854019-0da4ff7c-f705-11e7-8ea2-b32f22f9444b.png)

This is the design I went with in c13ea55897940569ff243b5bef20d480a2232b9e. I think it makes sense to obscure an option like that from the user so they don't accidentally press it. However, I removed the "delete" button from the Garden view entirely (unlike the screenshot) for now since it's not yet implemented. I left it in the profile view, but it just goes to an empty page.
